src/main/java/com/learn/assignment/Package.java

```diff
@@ -0,0 +1,63 @@
+package com.learn.assignment;
+
+import java.util.Objects;
+
+class Package implements Comparable<Package>{
+    String id;
+    double weight;
+    String city;
+
+    public Package(String id, double weight, String city) {
+        //System.out.println("------------In Package()--------------");
+        this.id = id;
+        this.weight = weight;
+        this.city = city;
+    }
+
+    public String getId() {
+        return id;
+    }
+
+    public double getWeight() {
+        return weight;
+    }
+
+    public String getCity() {
+        return city;
+    }
+
+    @Override
+    public String toString() {
+        //System.out.println("=========calling toString()=========");
```

Comment:
Why does this code need commented lines ?

Reply:
//System.out.println("=========calling toString()=========");
Removed the commented code.